src/components/BlogDetails/BlogDetails.scss

```diff
@@ -0,0 +1,54 @@
+@import "./../../styles/variables";
+
+.blog-details {
+  padding: 1rem;
+  width: 40%;
```

Comment:
maybe not needed?

Reply:
We need width for a better interface, like the mock-up , Vaiya; otherwise, it looks like
![image](https://github.com/mza-wtag/my-blog-react-app/assets/151821213/b5bef04d-5ec3-4cb4-8fee-c64bda90b8e6)